### PR TITLE
Test classify

### PR DESCRIPTION
**Pull request by manjumegh**

### CODE REVIEW

**Comment by rmarinho** [comment]
Hi @manjumegh what's missing is remove the comments  that were already classified from the potential-bad-01.csv . THe app should do that for you. When you click append results, it should show the following screen and you should say yes..

image

File: comments/classified.csv [hunk 1]
```diff
@@ -2511,6 +2511,212 @@ Yo what the fuck is your problem fucking around with my page! leave it the fuck
...
+* [x] Window Renamer / ID toast looks fine,0,0.5
+"* [x] About looks... Fine? The links aren't underlined anymore, but that might just be how WinUI works these days",0,0.5
+* [x] #codes look totally ~~insane~~ transparent,0,0.5
+. image](ht,0,0.5
+"I'm stupid, that should have been and #endif but is better with the NoXYZ attrs",0,0.5
+"Please excuse me if this is a stupid question, but why can't you just call the #code method here",0,0.5
+"Build what? Oh crap, I had built it from VS and it worked. I will take a look soon. 😥",0,0.5
+"That's fair, it could be made internal but it feels a little gross since users can new it up and use it in their own factory. Though that's gross as well",0,0.5
+"Crap - you're right... I didn't even think about that... yeah - that won't work. I'll try, but I'm pretty sure you're right",0,0.5
+gross,0,0.5
```
Comment (jonathanpeppers) [inline comments/classified.csv:2575]: I think this one should be negative:
```suggestion
gross,1,0.5
```

File: comments/classified.csv [hunk 2]
```diff
@@ -2511,6 +2511,212 @@ Yo what the fuck is your problem fucking around with my page! leave it the fuck
...
+- The code fails to compile at [rust-lang/rust#95508](https://github.com/rust-lang/rust/pull/95508),0,0.5
+- The bot ignores all context whatsoever at [moby/moby#43441](https://github.com/moby/moby/pull/43441),0,0.5
+"- After initial PR being closed, the bot keeps sending PRs, effectly spamming the repos. For example, these 5 PRs were sent to the same repo within 2 days [#1](https://github.com/996icu/996.ICU/pull/26073) [#2](https://github.com/996icu/996.ICU/pull/26074) [#3](https://github.com/996icu/996.ICU/pull/26075) [#4](https://github.com/996icu/996.ICU/pull/26076) [#5](https://github.com/996icu/996.ICU/pull/26077)",0,0.5
+"To save more time for the open-source community, we recommend the maintainers of VSCode to close this PR and ban this bot from further spamming",0,0.5
+Ignore failure due to https://github.com/xamarin/maccore/issues/2440 (AKA I'm an idiot),0,0.5
+i am a stupid person. typescript is garbage. Bill should be fired,0,0.5
+Changed back for now and put all the gross casting code back,0,0.5
```
Comment (jonathanpeppers) [inline comments/classified.csv:2584]: Both of these, I don't think they are trying to insult the PR author, but I think they should be negative:
```suggestion
i am a stupid person. typescript is garbage. Bill should be fired,1,0.5
Changed back for now and put all the gross casting code back,1,0.5
```
If they are using strong words, maybe we just mark it as negative on the safe side?

File: comments/classified.csv [hunk 3]
```diff
@@ -2511,6 +2511,212 @@ Yo what the fuck is your problem fucking around with my page! leave it the fuck
...
+"I was tempted to make a less terrible version, but there are two other ""copies"" of PresentError in appkit.cs and I wasn't sure what's worse",0,0.5
+#NAME?,0,0.5
+#NAME?,0,0.5
+"In either case, I can work on a XAMCORE_4_0 version for at least two, and do something (either match or be different) w\ this new one",0,0.5
+"@github - This is intended to optimize lab runs where we need to run build-test several times with limited sets of steps (e.g. native component build takes place on a different machine than managed component build and the layout gets generated on yet another machine in the lab pipeline) and Bruce noticed that we're unnecessarily duplicating some steps in such scenario. You're right that removing crossgenning of framework from ""generatelayoutonly"" is a breaking change, I'll fix that. Frankly speaking, the command-line syntax of build-test is a terrible mess with its arbitrary combination of exclusive (skip***) and inclusive (***only) commands",0,0.5
+Mono should change their formatter configuration if they don't want the includes sorted. Otherwise there's no way to enforce formatting,0,0.5
+How do I do that?  I'm an .editorconfig noob,0,0.5
+Gross,0,0.5
```
Comment (jonathanpeppers) [inline comments/classified.csv:2606]:
```suggestion
Gross,1,0.5
```

File: comments/classified.csv [hunk 4]
```diff
@@ -2511,6 +2511,212 @@ Yo what the fuck is your problem fucking around with my page! leave it the fuck
...
+"Furthermore, the types are cached in a map of strings to string literal types. Whenever testing the assignability of two string literal types, reference equality kicks in (which is fast)",0,0.5
+"Additionally, error messages are slightly better because you have a specific literal type to report (i.e. #code instead of #code)",0,0.5
+"@github I saw this was done for ObservableItemTemplateCollection for Windows, so I copied that. Mainly just for the dispatcher, but if we pass the view in, then we never have to worry about reaching out into the Application. or Device. worlds",0,0.5
+"Not sure if this is going to do terrible things with lifetime and/or memory. If so, we can change and find another way",1,0.5
+this seems like a terrible idea.  why not just compare to #code #Resolved,0,0.5
+Can we unify these codepaths,0,0.5
+"~I tried, but I couldn't come up with a good solution. I tried:~",0,0.5
+1. ~Take a SyntaxNode from the method to remove from the candidates~,0,0.5
+"~This does not work, because in VB you need go to the parent of the declaration and call #code on it:~",0,0.5
+~#code~,0,0.5
```
Comment (jonathanpeppers) [inline comments/classified.csv:2641]: If there is a really short line like this, you can use the skip button. Maybe remove this entire line:
```suggestion
```

**Review by jonathanpeppers — COMMENTED** [review]
Otherwise this looks good! These numbers seem to be improved from adding more data:
```
*       Average Class 0 Precision: 0.9  - Standard deviation: (.013)  - Confidence Interval 95%: (.013)
*       Average Class 1 Precision: 0.813  - Standard deviation: (.054)  - Confidence Interval 95%: (.053)
```

File: comments/classified.csv (hunk 1 above)
Comment (manjumegh) [reply comments/classified.csv]: looks like these comments are already been classified, this is at line 2216/2215

Comment (manjumegh) [reply comments/classified.csv]: So i updated the classified again and removed all redundant comments

**Pull request closed without merging**